packages/apps/graph-client/.env.example

```diff
@@ -0,0 +1,2 @@
+# Depth necessary on a block to call it "confirmed", omit to use default value: 6
+REACT_APP_MAX_CALCULATED_CONFIRMATION_DEPTH=
```

Comment:
Do we need the REACT_APP_ prefix?

Reply:
Since I need to expose these variables to the client (browser), I need it to be there (it does not work without the prefix). 
Also, the prefix in this file was incorrect. The correct one is `NEXT_PUBLIC`, and I already updated the fiel. This information is available on Next.js [documentation](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser)